packages/signature-controller/src/SignatureController.ts

```diff
@@ -250,6 +251,27 @@ export class SignatureController extends BaseControllerV2<
     return this.#typedMessageManager.getUnapprovedMessagesCount();
   }
 
+  /**
+   * Retrieves an array of all messages from TypedMessages and PersonalMessages.
+   *
+   * @returns The array of all messages from their proxies.
+   */
+  get allMessages(): unknown[] {
```

Comment:
For the return type, would it be helpful to instead return an object keyed on the message ID as each entry in the has it's own unique ID?

And rather than `unknown`, would it be more explicit to use `Message | PersonalMessage | TypedMessage`?

Reply:
Thank you! 🙏🏼